tomcat.yaml

```
@@ -0,0 +1,44 @@
+    become_user: root
+    lineinfile:
+      path: /usr/local/apache-tomcat-9.0.62/webapps/manager/META-INF/context.xml
+      # The String to Search
+      regexp: allow="127\.\d+\.\d+\.\d+|::1|0:0:0:0:0:0:0:1" />
+      # The String to Replace
+      line: allow="\d+\.\d+\.\d+\.\d+" />
+      state: present
+      backup: yes
+      
+  - name: start tomcat services
+    shell: nohup /usr/local/apache-tomcat-9.0.62/bin/startup.sh
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)